web/styles/settings.css

```diff
@@ -544,6 +544,31 @@ input[type="checkbox"] {
     word-break: break-word;
 }
 
+.alert-notification-email {
+    display: inline-block;
+    vertical-align: top;
+    height: auto !important;
+    width: auto !important;
+    white-space: break-spaces !important;
```

Comment:
Please use a comment whenever you use `!important` explaining why you needed it... or use a higher precedence selector or classes to avoid needing it.